xknx/exceptions/exception.py

```diff
@@ -196,3 +196,7 @@ class ManagementConnectionRefused(ManagementConnectionError):
 
 class ManagementConnectionTimeout(ManagementConnectionError):
     """Exception class used when a management connection timed out."""
+
+
+class ManagementConnectionWriteAddressError(ManagementConnectionError):
```

Comment:
Do you need this granularity of exceptions? Otherwise we could just raise ManagementConnectionError and use the message to tell the user what happened.

I'm fine either way, just curious.

Reply:
No directly, so I could refactor it out if you prefer. I just wanted to add some additional level of detail.